_posts/2021-03-23-connector-build.md

```diff
@@ -0,0 +1,232 @@
+---
+layout: post
+title:  "How to build your own KafkaConnect image via custom resource"
+date: 2021-03-23
+author: jakub_stejskal
+---
+In one of [the older blog post](https://strimzi.io/blog/2020/05/07/camel-kafka-connectors/) we shown you how easy is to integrate Camel with Strimzi via KafkaConnect custom resource.
+That approach had one limitation - you had to build your own KafkaConnect image and use it in custom resource.
+However, this step is not needed anymore, thanks to feature which we introduced in Strimzi 0.22 to allow users to build custom KafkaConnect images directly by Strimzi.
+
+### How does it work
+Strimzi uses two different ways how to build the image based on the underlying kubernetes cluster:
+* In case you use pure Kubernetes, the image will be built by [Kaniko](https://github.com/GoogleContainerTools/kaniko)
+* In case you use Openshift, the image will be built by Openshift Builds
+
+In both cases, Strimzi will spin-up a build pod, which builds the image based on the configuration from custom resource.
+The final image is then pushed into specific registry or image stream, again based on the configuration. 
+The great thing in this feature is, that you can easily reuse the newly built image in another KafkaConnect resource without building it again or manually.
```

Comment:
This sounds to me like following:
* I create `KafkaConnect` cluster `my-connect` with the `build` section which builds the image
* Then I create somewhere another `KafkaConnect` resource but instead of specifying the `build` section again I just use `image` to point to the image from the other cluster.

But this is not how it is intended to work. I think you should make it more clear here that the image will be built and used for the same Connect cluster.

Reply:
Understood, I made some changes to last sentence.